### PR TITLE
Implement recursive noise dataset loading

### PR DESCRIPTION
**Pull request by daemon**
- Make noise dataset load all **/*.wav files in the specified folder

### CODE REVIEW

File: ww4ff/data/transform/augment.py [hunk 1]
```diff
@@ -154,26 +160,36 @@ def augment(self, param, examples: Sequence[EmplacableExample], **kwargs):
...
+            b = self.rand.randint(waveform.size(-1), bg_ex.size(-1))
             a = b - waveform.size(-1)
             bg_audio = bg_ex[..., a:b]
-            alpha = random.random() * param.magnitude
+            alpha = 1 if param.name == 'replace' else self.rand.random() * param.magnitude
             mixed_wf = waveform * (1 - alpha) + bg_audio * alpha
-            new_examples.append(example.emplaced_audio_data(mixed_wf))
+            ex = example.emplaced_audio_data(mixed_wf)
+            if alpha == 1:
+                ex.frame_labels = {}
```
Comment (ljj7975) [inline ww4ff/data/transform/augment.py:191]: So if replace, it just replace existing audio sample to be a noise sample and update frame_label accordingly?, Seems like the ww label is not being updated

File: ww4ff/data/transform/augment.py [hunk 2]
```diff
@@ -124,7 +130,7 @@ def default_params(self):
     def augment(self, param, examples: Sequence[EmplacableExample], **kwargs):
         new_examples = []
         for example in examples:
-            rate = np.clip(np.random.normal(1, param.magnitude), 0.5, 1.5)
+            rate = np.clip(np.random.normal(1.1, param.magnitude), 0.8, 2)
```
Comment (ljj7975) [inline ww4ff/data/transform/augment.py:133]: stretching audio more?

Comment (daemon) [reply ww4ff/data/transform/augment.py:133]: Speeding it up more.

File: ww4ff/data/transform/augment.py (hunk 1 above)
Comment (daemon) [reply ww4ff/data/transform/augment.py:191]: The batchifier uses only `frame_labels` I believe.

**Pull request merged**